build.ps1

```diff
@@ -59,14 +61,14 @@ if ($RealSign) {
     $signType = 'real'
 }
 
-$commonBuildArgs = echo $RepoRoot\build\build.proj /m /nologo /p:Configuration=$Configuration /p:Platform=$Platform /p:SignType=$signType 
+$commonBuildArgs = echo $RepoRoot\build\build.proj /m /nologo /p:Configuration=$Configuration /p:Platform=$Platform /p:SignType=$signType /p:Verbosity=$Verbosity
```

Comment:
This is setting an MSBuild property named `Verbosity` instead of setting the parameter to each of the loggers.  I don't think setting a `Verbosity` property will have any effect.  I think you still need to pass in the verbosity separately to each logger.